src/ApparentHorizons/StrahlkorperDataBox.cpp

```diff
@@ -0,0 +1,252 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "ApparentHorizons/StrahlkorperDataBox.hpp"
+
+#include "DataStructures/MakeWithValue.hpp"
+
+namespace StrahlkorperTags {
+
+template <typename Frame>
+typename types<Frame>::ThetaPhi ThetaPhi<Frame>::function(
```

Comment:
You can replace all the return types with `db::item_type<TAG>`, specifically here `typename types<Frame>::ThetaPhi` -> `db::item_type<ThetaPhi<Frame>>`

Reply:
Hmm. If I try this, compiler complains that `types<Frame>::ThetaPhi ` (in the declaration) is a different type than `db::item_type<ThetaPhi<Frame>>` (in the definition).  And if I put `db::item_type<ThetaPhi<Frame>>` as the return type in the declaration I get even more problems (e.g. incomplete type).

Reply:
Yes, it won't work in the header file. @wthrowe and I can discuss tomorrow how we think we should proceed on this and post a reply